dead-simple-k8s/hostIP.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+set -e
+
+sleep 10
+curl -s "${K8S_API:-localhost:8001}/api/v1/namespaces/$NS/pods/$POD_NAME" | jq '.status.hostIP' | sed 's/"//g'
```

Comment:
Seems like a tall order to ask that jq be installed (this won't ever work on busybox etc)...